### PR TITLE
provide the section 'reports' in the metadata filed instead of the parameter 'print_report'

### PR DESCRIPTION
**Pull request by Anna050689**
provide the section 'reports' in the metadata file instead of the parameter 'print_report'

### CODE REVIEW

File: src/syngen/infer.py [hunk 1]
```diff
@@ -48,11 +56,18 @@
...
-    type=click.BOOL,
-    help="Whether to print quality report. Might require significant time "
-    "for big generated tables (>1000 rows). If absent, it's defaulted to False",
+    "--reports",
+    default=("none",),
+    type=click.UNPROCESSED,
+    multiple=True,
+    callback=validate_reports,
+    help="Controls the generation of quality reports. "
+    "Might require significant time for big generated tables (>1000 rows). "
```
[the remark this answers is not in the document]
Comment (Anna050689) [reply src/syngen/infer.py:65]: I agree with you. I will change it in the next commit.

File: README.md [hunk 2]
```diff
@@ -103,7 +112,7 @@ Requirements for parameters of training process:
 * <i>row_limit</i> - data type - integer
 * <i>drop_null</i> - data type - boolean, default value - False
 * <i>batch_size</i> - data type - integer, must be equal to or more than 1, default value - 32
-* <i>print_report</i> - data type - boolean, default value is False
+* <i>reports</i> - data type - if the value is passed through CLI - string, if the value is passed in the metadata file - string (if value is "none" or "all") or list, accepted values: <i>"none"</i> (default) - no reports will be generated, <i>"all"</i> - generates both accuracy and sampling reports, <i>"accuracy"</i> - generates an accuracy report, <i>"sample"</i> - generates a sampling report, <i>"metrics_only"</i> - outputs the metrics information only to standard output. In the metadata file multiple values can be specified as a list to generate multiple types of reports simultaneously
```
[the remark this answers is not in the document]
Comment (Anna050689) [reply README.md:115]: I agree with you. I will add this information in the next commit.

File: README.md [hunk 3]
```diff
@@ -103,7 +112,7 @@ Requirements for parameters of training process:
 * <i>row_limit</i> - data type - integer
 * <i>drop_null</i> - data type - boolean, default value - False
 * <i>batch_size</i> - data type - integer, must be equal to or more than 1, default value - 32
-* <i>print_report</i> - data type - boolean, default value is False
+* <i>reports</i> - data type - if the value is passed through CLI - string, if the value is passed in the metadata file - string (if value is <i>"none"</i> or <i>"all"</i>) or list, accepted values: <i>"none"</i> (default) - no reports will be generated, <i>"all"</i> - generates both accuracy and sample reports, <i>"accuracy"</i> - generates an accuracy report, <i>"sample"</i> - generates a sample report, <i>"metrics_only"</i> - outputs the metrics information only to standard output. Default value is <i>none</i>. In the metadata file multiple values can be specified as a list to generate multiple types of reports simultaneously.
```
Comment (rioman) [inline README.md:115]: May be add that value "metrics_only" doesn't generate reports in files?

File: README.md [hunk 4]
```diff
@@ -142,13 +151,13 @@ Requirements for parameters of generation process:
 * <i>run_parallel</i> - data type - boolean, default value is False
 * <i>batch_size</i> - data type - integer, must be equal to or more than 1
 * <i>random_seed</i> - data type - integer, must be equal to or more than 0
-* <i>print_report</i> - data type - boolean, default value is False
+* <i>reports</i> - data type - if the value is passed through CLI - string, if the value is passed in the metadata file - string (if value is <i>"none"</i> or <i>"all"</i>) or list, accepted values: <i>"none"</i> (default) - no reports will be generated, <i>"all"</i> - generates an accuracy report, <i>"accuracy"</i> - generates an accuracy report, <i>"metrics_only"</i> - outputs the metrics information only to standard output. Default value is <i>none</i>. In the metadata file multiple values can be specified as a list to generate multiple types of reports simultaneously
```
Comment (rioman) [inline README.md:154]: Sorry, doesn't know how to pass list in yaml files. Just a comma separated list? Or separate entries with the same name and different values?

File: README.md (hunk 3 above)
Comment (Anna050689) [reply README.md:115]: I will add this information.

**Pull request merged**